swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -284,14 +308,21 @@ module Stmts {
 
     class LoopStmt = @for_each_stmt or ConditionalLoop;
 
-    abstract class LoopTree extends PreOrderTree instanceof ConditionalLoop {
-      abstract AstNode getCondition();
+    abstract class LoopTree extends AstPreOrderTree {
+      LoopTree() {
+        ast instanceof WhileStmt or
```

Comment:
why not `ast instanceof ConditionalLoop`?

Reply:
Good point. I'll fix that in a follow-up in a second.

Reply:
PR with a fix: https://github.com/github/codeql/pull/9329